docs/vectorizer-quick-start.md

```diff
@@ -43,6 +43,13 @@ On your local machine:
     docker compose up -d
     ```
 
+1. **Download the Ollama models.** We'll use the `nomic-embed-text` model to generate embeddings.
+
+    ```
+    docker compose exec ollama ollama pull nomic-embed-text
+    ```
+
+
```

Comment:
James removed the pull intentionally here:
https://github.com/timescale/pgai/pull/301

Was that a mistake, is that feature broken? Or is it the ai.embed call that needs it? Then again the vectorizer should pull the model for you.

Reply:
I see, but the example is broken without that @Askir

Reply:
we can evolve this to be something else but as of today we need to pull them in order to make the example work.